subsys/bluetooth/audio/ascs.c

```diff
@@ -1518,6 +1518,49 @@ struct ascs_parse_result {
 	size_t count;
 };
 
+static bool ascs_valid_metadata_type(uint8_t type, uint8_t len)
+{
+	switch (type) {
+	case BT_AUDIO_METADATA_TYPE_PREF_CONTEXT:
+	case BT_AUDIO_METADATA_TYPE_STREAM_CONTEXT:
```

Comment:
Is it required to mark the fallthroughs?  (I do not see it as necessary here, but I see it done elsewhere.)n

Reply:
Fallthrough is necessary if the case is doing something, and then falling through, e.g.

```c
case 'a':
    i++;
    /* fall through */
case 'b':
    j++;
    break;
case 'c':
case 'd':
    k++;
    break;
```